cmd/root.go

```diff
@@ -123,16 +124,17 @@ func init() {
 		viper.BindPFlag("verbose-level", flag.CommandLine.Lookup("v"))
 	}
 
-	// Register subcommands.
-	rootCmd.AddCommand(apidiff.Cmd)
-	rootCmd.AddCommand(apidump.Cmd)
-	rootCmd.AddCommand(apispec.Cmd)
+	// Register subcommands. Most commands that interact with Akita Cloud should
+	// be guarded by ci_guard.
+	rootCmd.AddCommand(ci_guard.GuardCommand(apidiff.Cmd))
```

Comment:
I'm not sure why apidiff is on the list?  (I'm not sure it would be used in a CI run anwyay, so it doesn't matter.)

Reply:
It interacts with Akita Cloud and consumes cloud resources to produce the diff. It's also not a purely interactive command: you can write the diff to a file with `--out`.